<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent">
     <property name="title">
      <string>Open Recent</string>
     </property>
    </widget>
    <addaction name="actionNew_Context"/>
    <addaction name="actionNew_Note"/>
    <addaction name="menuOpen_Recent"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport_Context_as_File"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuNote">
    <property name="title">
     <string>Note</string>
    </property>
    <addaction name="actionMove_to_archive"/>
    <addaction name="actionShow_hide_archived_notes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuNote"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Import From File...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNew_Context">
   <property name="text">
    <string>New Context...</string>
   </property>
  </action>
  <action name="actionNew_Note">
   <property name="text">
    <string>New Note...</string>
   </property>
  </action>
  <action name="actionMove_to_archive">
   <property name="text">
    <string>Move to archive</string>
   </property>
  </action>
  <action name="actionShow_hide_archived_notes">
   <property name="text">
    <string>Show/hide archived notes</string>
   </property>
  </action>
  <action name="actionrecent">
   <property name="text">
    <string>recent</string>
   </property>
  </action>
  <action name="actionExport_Context_as_File">
   <property name="text">
    <string>Export Context as File...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
